src/modules/Nis.rb

```diff
@@ -908,10 +908,8 @@ def Read
       @multidomain_broadcast = {} if @multidomain_broadcast == nil
       @slp_domain = {} if @slp_domain == nil
 
-      out = Convert.to_map(
-        SCR.Execute(path(".target.bash_output"), "/bin/ypdomainname")
-      )
-      @domain = Builtins.deletechars(Ops.get_string(out, "stdout", ""), "\n")
+      out = SCR.Execute(path(".target.bash_output"), "/usr/bin/ypdomainname")
+      @domain = out["stdout"].strip
```

Comment:
do you really want to change behavior to strip? Previous behavior looks like chomp http://www.ruby-doc.org/core-2.2.0/String.html#method-i-chomp on opossite strip remove all whitespaces http://www.ruby-doc.org/core-2.2.0/String.html#method-i-strip